contributing/TESTING.md

```diff
@@ -159,7 +159,7 @@ There are 5 serving modes:
 - RTV mode serves the bundle from the given RTV number (a 15 digit number). E.g. `001907161745080`. Serve RTV mode by running `gulp serve --rtv <rtv_number>`
 - ESM mode serves the esm (module) binaries. First run `gulp dist --fortesting --esm` and then serve esm mode by running `gulp serve --new_server --esm`. _This mode is new, and under active development._
 
-To switch serving mode during runtime, go to http://localhost:8000/serve_mode=$mode and set the `$mode` to one of the following values: `default`, `compiled`, `cdn` or `<RTV_NUMBER>`.
+To switch serving mode during runtime, go to http://localhost:8000/serve_mode=\$mode and set the `$mode` to one of the following values: `default`, `compiled`, `cdn` or `<RTV_NUMBER>`.
```

Comment:
Is this true? `\$mode` versus `$mode`?

Reply:
Turns out the original URL needn't have had the `$` in the first place. I've fixed and verified the rendered diffs look okay.

![image](https://user-images.githubusercontent.com/26553114/96305960-bc30ec80-0fcc-11eb-8eee-8e1047492fa4.png)